plugin/editorconfig.vim

```diff
@@ -232,47 +234,17 @@ function! s:InitializePythonBuiltin(editorconfig_core_py_dir) " {{{2
     let s:builtin_python_initialized = 1
 
     let l:ret = 0
-
-    if !has('python')
-        return 1
-    endif
-
-    python << EEOOFF
-
-try:
-    import vim
-    import sys
-except:
-    vim.command('let l:ret = 2')
-
-EEOOFF
-
-    if l:ret != 0
-        return l:ret
-    endif
-
-    python << EEOOFF
-
-try:
-    sys.path.insert(0, vim.eval('a:editorconfig_core_py_dir'))
-
-    import editorconfig
-    import editorconfig.exceptions as editorconfig_except
-
-except:
-    vim.command('let l:ret = 3')
-
-del sys.path[0]
-
-ec_data = {}  # used in order to keep clean Python namespace
-
-EEOOFF
-
-    if l:ret != 0
-        return l:ret
+    if has('python')
+        execute 'pyfile ' . s:pyscript_path
+        let s:py_eval = 'pyeval'
+    elseif has ('python3')
+        execute 'py3file ' . s:pyscript_path
+        let s:py_eval = 'py3eval'
```

Comment:
Though I'm not sure whether it can be better or useful, I think it is okay. Anyway this change is easy and I suppose there is no problem.